msg/en_US/utils.msg

```diff
@@ -986,6 +986,7 @@ valid options:\n\
   -p, --page=ID               page id; the -L option is required; default : 0\n\
   -v, --verbose               enable verbose status messages; default : disable\n\
   -i, --interval=S            print information every S secs\n
+117 Query result is failed to load. First, wrong directory of log file makes this error. Second, deletion db_ha_apply_info infomation makes this error. \n
```

Comment:
Please, make consistency with other utilities.

Reply:
Did you mean that the prefix number '117' and not continous number?

I knew the number of prefix should be related with same number.
For example, 117 is referenced from `LOADDB_MSG_LOAD_FAIL = 117`, and  no. 60 usually means usage and all the number is not continuous.